reference/5.1/PSDesiredStateConfiguration/Get-DscLocalConfigurationManager.md

```diff
@@ -100,7 +173,11 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: **-Debug**, **-ErrorAction**, **-ErrorVariable**,
```

Comment:
This is an exception in our formatting, Normally, these should be in backticks (instead of bold) since they have the dashes. However, there are two problems.

1. This text is autogenerated by PlatyPS and gets overwritten when updates are run
2. This text is not rendered by OPS anyway (I have an open bug for this)

So not bother with markup. Just fix proper line spacing and length wrapping.

Reply:
I though so, I was almost positive they were supposed to be in backticks and then I was going through the checklist and decided to play it safe. another rake in the face. Fixed